content/03-reference/01-tools-and-interfaces/02-prisma-client/055-case-sensitivity.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: 'Case sensitivity'
+metaTitle: 'Case sensitivity (Reference)'
+metaDescription: 'How Prisma Client handles case sensitivity when filtering and sorting.'
+preview: false
+---
+
+<TopBlock>
+
+Case sensitivity affects **filtering** and **sorting** of data, and is determined by your [database collation](#database-collation-and-case-sensitivity). Sorting and filtering data yields different results depending on your settings:
+
+| Action          | Case sensitive                               | Case insensitive                             |
+|-----------------|----------------------------------------------|----------------------------------------------|
+| Sort ascending  | `Apple`, `Banana`, `apple pie`, `banana pie` | `Apple`, `apple pie`, `Banana`, `banana pie` |
+| Match `"apple"` | `apple`                                      | `Apple`, `apple`                             |
+
+[Prisma Client](.) respects your database collation. Options and recommendations for supporting **case-insensitive** filtering and sorting with Prisma Client depend on your [database provider](#options-for-case-insensitive-filtering).
```

Comment:
`(.)` is that a missing link? or is it valid?

Reply:
It's valid - `..` did not work, oddly, but `.` did